subway_structure.py

```diff
@@ -1780,36 +1786,40 @@ def validate(self):
                 msg = 'Found {} stations in routes, expected {}'.format(
                     self.found_stations, self.num_stations
                 )
-                self.error_if(
-                    not (
-                        0
-                        <= (self.num_stations - self.found_stations)
-                        / self.num_stations
-                        <= ALLOWED_STATIONS_MISMATCH
-                    ),
-                    msg,
+                log_function = (
```

Comment:
Why not put this tructure into `error_if`?

Reply:
With three error levels more combinations arise. Now at least `error_or_warning_if` and `error_or_notice_if` functions would be needed. I removed `error_if` function at all.